parentchain/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+
+#[frame_support::pallet]
+pub mod pallet {
+	use crate::weights::WeightInfo;
+	use frame_support::pallet_prelude::*;
+	use frame_system::pallet_prelude::*;
+
+	#[pallet::pallet]
+	#[pallet::generate_store(pub(super) trait Store)]
+	pub struct Pallet<T>(_);
+
+	/// Configuration trait.
+	#[pallet::config]
+	pub trait Config: frame_system::Config {
+		type WeightInfo: WeightInfo;
+	}
+
+	/// The current block number being processed. Set by `execute_block`.
+	#[pallet::storage]
+	#[pallet::getter(fn block_number)]
+	pub(super) type Number<T: Config> = StorageValue<_, T::BlockNumber, ValueQuery>;
```

Comment:
I would like if  the storage type and the getter always have the same name. Here, it'd be: `BlockNumber`.

Reply:
Are you sure about that @clangenb ? The frame_system pallet also uses `Number` (https://github.com/paritytech/substrate/blob/master/frame/system/src/lib.rs#L607-L610). As this pallet is a simplified system pallet, should we not use the same naming conventions?

Reply:
Ok, I agree that we should not change conventions here. I am just very surprised by the conventions that substrate uses in that case. :) fine with we, let's leave it.